about.html

```diff
@@ -0,0 +1,168 @@
+      <section
+        class="bg-white px-[10%] text-center text-black flex flex-col items-center gap-3"
+      >
+        <h2 class="text-center text-[#272a31] text-[25px] mt-[5%]">
+          The ultimate wine tasting party of 2023
+        </h2>
+        <hr class="w-[40px] bg-[#ec5242] h-[3px]" />
+        <p class="text-center">
+          The logo of CC Global Summit 2023 was decided through the logo
+          competition from 8, June to 7, July.
+        </p>
+        <div
+          class="my-[5%] px-[5%] py-[2%] border-solid border-[1px] border-[#d3d3d3]"
+        >
+          <img src="assets/wine-glass.png" alt="" />
+        </div>
+      </section>
+
+      <!-- past events -->
+
+      <hr class="bg-[#ec5242] flex w-[100%] item-center" />
+      <section
+        class="px-[10%] flex flex-col items-center gap-3 justify-center items-center"
+      >
+        <h2 class="text-center text-[#272a31] text-[25px] mt-[5%]">
+          See the past Wine tasting Parties
+        </h2>
+        <hr class="w-[40px] bg-[#ec5242] h-[3px]" />
+        <p>
+          Take a look at the past two Wine tasting parties which took place in
+          Lesotho and in Botswana
+        </p>
+        <div
+          class="ml-[3%] mt-[50px] pb-[100px] grid gap-[10px] grid-cols-1 md:grid-cols-2 justify-items-center"
+        >
+          <div
+            class="h-[320px] bg-[url('assets/hero-background.jpg')] bg-contain"
+          >
+            <div
+              class="text-[#fff] hover:bg-[#ec53423d] bg-[#ec5342bd] h-[inherit] p-[20px] flex justify-center flex-col flex-nowrap gap-5 items-center"
+            >
+              <h2 class="text-[25px] text-[#fff]">2019</h2>
+              <p class="text-center">Wine Tasting Party in Lesotho.</p>
+            </div>
+          </div>
+
+          <div
+            class="h-[320px] bg-[url('assets/hero-background.jpg')] bg-contain"
+          >
+            <div
+              class="text-[#fff] hover:bg-[#ec53423d] bg-[#ec5342bd] h-[inherit] p-[20px] flex justify-center flex-col flex-nowrap gap-5 items-center"
+            >
+              <h2 class="text-[25px] text-[#fff]">2021</h2>
+              <p class="text-center">Wine Tasting Party in Botswana.</p>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <!-- footer -->
+
+    <footer class="px-[10%] py-[5%] text-white flex bg-[#414246]">
+      <a href=""><img src="assets/wine-glass.png" alt="" />wine~up</a>
+      <div class="px-[5%]">
+        <h3>2023 Naledi Dikgale South Africa. some rights reserved</h3>
+        <p class="hidden md:block">
+          Naledi Dikgale copyright “all rights reserved” 달리 정하지 않은
+          웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국
+          라이선스에 따라 이용할 수 있습니다. 자세한 이용기능 내용과 허락조건은
+          http://naldi22khosi@gmail.com/licences/by/2.0/kr 를 보세요.
+        </p>
+      </div>
+
+    </footer>
+
+    <script src="script.js"></script>
+  </body>
+</html>
```

Comment:
The footer has a white background in the provided design compared to black in your case. Kindly fix this issue.  I have recorded [this video](https://www.loom.com/share/995ea0b1c5eb4346a5f0d096bb8e3b5b) where you can verify the problem.